screens/UsersScreen/index.js

```diff
@@ -39,29 +32,23 @@ const UsersScreen = ({navigation}) => {
             }
```

Comment:
create an imageUrl variable and move this condition there

Reply:
Done !